### DISCUSSION TITLE
Installation fails using Installation Script "URL http://127.0.0.1:8888 not available, check searx & uwsgi setup!"

### DISCUSSION THREAD

**Discussion by ioogithub** [searx/searx#2878] [Installation issue: Official wiki]
**Version**
```
commit c5d63a5c9718b18b738856dad90a140cc1b51b55 (HEAD -> master, origin/master, origin/HEAD)
Merge: 2db2dfa8 9e7a6848
Author: Noémi Ványi <kvch@users.noreply.github.com>
Date:   Thu Jun 3 22:17:29 2021 +0200

    Merge pull request #2837 from searx/update_data_update_currencies.py
    
    Update searx.data - update_currencies.py
``` 

**How did you install Searx?**
Installed using official wiki install script.

`sudo -H ./utils/searx.sh install all
`
**What happened?**
The script runs run until I get the error that it cannot find a file then exits.

**How To Reproduce**
1. Follow the instructions here: https://searx.github.io/searx/admin/installation.html#installation-scripts
2. Run $ sudo -H ./utils/searx.sh install all

Script generates errors:
```
Install searx's uWSGI app (searx.ini)
-------------------------------------

INFO:  installing uwsgi ...

installation of packages
------------------------

package(s)::

Should packages be installed? [YES/no] 
Yes
INFO:  prepare (searx:searx)  /run/uwsgi/app/searx
mkdir: cannot create directory ‘’: No such file or directory
INFO:  install (eval=1): /searx.ini
ERROR: /path/to/file/searx/utils/templates/searx.ini does not exists
ERROR: ... can't install /searx.ini
** press any [KEY] to continue **
ERROR: neon-20.04: uWSGI not yet implemented
INFO:  restart uWSGI service
ERROR: neon-20.04: uWSGI not yet implemented
INFO:  uWSGI app: searx.ini is installed
ERROR: URL http://127.0.0.1:8888 not available, check searx & uwsgi setup!
```

**Expectations**
I expect the install script will complete successfully and searx will be functional.

**Additional context**
The file it is looking for does not exist. In step 1 I cloned the installation as described in the wiki so all files that are needed by the installation should be avaliable.

**System**
KDE Neon version 5.22 (Ubuntu)

**Comment by 0xhtml** [comment]
Try installing `python-is-python3` and `python3-pip` first.
e.g.: `sudo apt install python-is-python3 python3-pip`

**Comment by IndiTheo** [comment]
I have the same issue on Manjaro Linux 0.6.6 x86_64
